inference-engine/src/preprocessing/ie_preprocess_gapi.cpp

```diff
@@ -169,9 +169,50 @@ std::vector<std::vector<cv::gapi::own::Mat>> bind_to_blob(const I420Blob::Ptr& i
     std::vector<std::vector<cv::gapi::own::Mat>> batched_input_plane_mats(batch_size);
     for (int i = 0; i < batch_size; ++i) {
         auto& input = batched_input_plane_mats[i];
-        input.emplace_back(std::move(batched_y_plane_mats[i][0]));
-        input.emplace_back(std::move(batched_u_plane_mats[i][0]));
-        input.emplace_back(std::move(batched_v_plane_mats[i][0]));
+        input.emplace_back(std::move(batched_y_plane_mats[i][0])); //[0] cause only one channel for y
+        input.emplace_back(std::move(batched_u_plane_mats[i][0])); //[0] cause only one channel for u
+        input.emplace_back(std::move(batched_v_plane_mats[i][0])); //[0] cause only one channel for v
+    }
+
+    return batched_input_plane_mats;
+}
+
+std::vector<std::vector<cv::gapi::own::Mat>> bind_to_blob(const BatchedBlob::Ptr& blob,
+                                                          int batch_size) {
+    int underlied_blob_size = 1;
```

Comment:
-> `underlying`? 

Also, why it is 1?

Reply:
should it be `const`?

Reply:
as far I remember - it means, that batched blob aggregates N underlie blobs which actually represent 1 blob (not batched).
It according to BatchedBlob design: no stacked batched blob inside - otherwise BacthedBlob constructor calls abort() 
From the other side NV12 & I420 blobs (inside BatchedBlob) has managed to contains only single plane but  MemoryBlob can contains multiple dimension(?). so initially we assume that `underlied_blob_size  == 1` ( assume NV12 or I420) but then we check actual inner blob type  and reallocate `underlied_blob_size ` in case of MemoryBlob with non single-plane

Reply:
This comment is helpful so I think it should be there, thanks